packages/adapter-libsql/src/conversion.ts

```diff
@@ -134,12 +134,8 @@ export function mapRow(row: Row, columnTypes: ColumnType[]): ResultValue[] {
   for (let i = 0; i < row.length; i++) {
     const value = row[i]
 
-    // Convert array buffers to arrays of bytes.
-    // Base64 would've been more efficient but would collide with the existing
-    // logic that treats string values of type Bytes as raw UTF-8 bytes that was
-    // implemented for other adapters.
     if (value instanceof ArrayBuffer) {
```

Comment:
See https://github.com/prisma/prisma/pull/28913/changes#r2619173013, which might be relevant here as well.